metacontrol_experiments.rosinstall

```diff
@@ -38,8 +38,8 @@
     version: master
 - git:
     local-name: metacontrol_experiments
-    uri: https://github.com/rosin-project/metacontrol_experiments.git
-    version: master
+    uri: https://github.com/ipa-hsd/metacontrol_experiments.git
```

Comment:
Should this point to the rosin repo now? 

I'm thinking maybe we can leave it and change it later

Reply:
Actually we can even remove this, as `uses: actions/checkout@v2` gets it

Reply:
I am creating a new workspace and not using the default `actions/checkout` workspace. I will change `ipa-hsd` to `rosin-project` for now.